release-notes/release-notes/5.8-release-notes.md

```diff
@@ -57,6 +61,8 @@ You can now pull specific data from REST API response headers using .NET regex c
 - We fixed a bug in CQL on PostgreSQL that caused the `DATEADD` function to truncate input dates down to `DAY` precision. Now, you can expect more accurate date manipulations without losing finer time details.
 - We fixed an issue where the UI failed to display Batch Data Sync results and instead showed a generic exception message. The jobs tab in the UI now opens without any API failure appearing in the browser's network console.
 - We resolved an issue that caused large batch delta syncs to fail in Cinchy. 
+- We fixed an issue where Cinchy CDC Delete events weren't sent to the destination using Delta. **Deletes** and **Approved Deletes** now successfully insert records into Kafka when deleted from a Cinchy table. 
+- We fixed the issue of concurrent updates failing due to a Primary Key (PK) violation on the History table by adding a retry mechanism. This fix aims to make Cinchy more robust when making concurrent updates.
```

Comment:
CIN-06665. Not much information in the ticket. Not sure if we want to call attention to making it more 'robust' or 'resilient'.